src/styles/blocks/names.scss

```diff
@@ -0,0 +1,14 @@
+.name {
+  display: flex;
+  gap: #{$gap};
+  margin-bottom: 40px;

```

Comment:
It is forbidden to set margins for blocks, but for block's elements are allowed. The block should stretch to accommodate its children